lib/ansible/modules/cloud/cloudstack/cs_vpc_offering.py

```diff
@@ -0,0 +1,290 @@
+    - Create, update, enable, disable and remove CloudStack VPC offerings.
+version_added: '2.5'
+author: "David Passante (@dpassante)"
+options:
+  name:
+    description:
+      - The name of the vpc offering
+    required: true
+  state:
+    description:
+      - State of the vpc offering.
+    choices: [ enabled, present, disabled, absent ]
+    required: false
+    default: present
+  display_text:
+    description:
+      - Display text of the vpc offerings
+    required: false
+  service_capabilities:
+    description:
+      - Desired service capabilities as part of vpc offering.
+    aliases: [ service_capability ]
+  service_offering:
+    description:
+      - The name or ID of the service offering for the VPC router appliance.
+    required: false
+  supported_services:
+    description:
+      - Services supported by the vpc offering
+    aliases: [ supported_service ]
+    required: false
+  service_providers:
+    description:
+      - provider to service mapping. If not specified, the provider for the service will be mapped to the default provider on the physical network
+    aliases: [ service_provider ]
+    required: false
+  poll_async:
+    description:
+      - Poll async jobs until job has finished.
+    default: true
+extends_documentation_fragment: cloudstack
+'''
+
+EXAMPLES = '''
+# Create a vpc offering and enable it
+- local_action:
+    module: cs_vpc_offering
+    name: "my_vpc_offering"
+    display_text: "vpc offering description"
+    state: enabled
+    supported_services: [ Dns, Dhcp ]
+    service_providers:
+      - {service: 'dns', provider: 'virtualrouter'}
+      - {service: 'dhcp', provider: 'virtualrouter'}
+
+# Remove a vpc offering
+- local_action:
+    module: cs_vpc_offering
+    name: "my_vpc_offering"
+    display_text: "vpc offering description"
+    state: absent
+    supported_services: [ Dns, Dhcp ]
+    service_providers:
+      - {service: 'dns', provider: 'virtualrouter'}
+      - {service: 'dhcp', provider: 'virtualrouter'}
+'''
+
+RETURN = '''
+---
+id:
+  description: UUID of the vpc offering.
+  returned: success
+  type: string
+  sample: a6f7a5fc-43f8-11e5-a151-feff819cdc9f
+name:
+  description: The name of the vpc offering
+  returned: success
+  type: string
+  sample: MyCustomVPCOffering
+display_text:
+  description: The display text of the vpc offering
+  returned: success
+  type: string
+  sample: My vpc offering
+state:
+  description: The state of the vpc offering
+  returned: success
+  type: string
+  sample: Enabled
+service_offering_id:
+  description: The service offering ID.
+  returned: success
+  type: string
+  sample: c5f7a5fc-43f8-11e5-a151-feff819cdc9f
+is_default:
+  description: Whether VPC offering is the default offering or not.
+  returned: success
+  type: bool
+  sample: false
+supportsregionLevelvpc:
+  description: Indicated if the offering can support region level vpc.
+  returned: success
+  type: bool
+  sample: false
+distributedvpcrouter:
```

Comment:
this should be `distributed` according to code

Reply:
You're right...